db/migrations/20200616091329-application-history.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
```

Comment:
I think this is almost identical to the file above except for the names of the up and down SQL files. Could we maybe extract this out to methods which return the up and down methods when called with the SQL filename?

Reply:
Ignore this @neilkidd. I see they are automatically created. I think they could be much leaner files, but that would probably involve a PR to `db-migrate` and more time than we have to play with on this project.